scripts/Update-QDKVersion.ps1

```diff
@@ -53,7 +53,7 @@ $dockerPath = Join-Path $katasRoot "Dockerfile"
     } | Set-Content -Path $dockerPath
 
 
-$ps1String = "Microsoft.Quantum.IQSharp --version $versionRegex"
+$ps1String = "Microsoft.Quantum.IQSharp --version 0.18.2107153439
```

Comment:
Please don't hardcode the new version in this script. Providing the version as a script parameter makes sure the script is reusable, and that the changes are done in all the right places. This line should be unchanged, but line 21 in this script should be updated, and the file `.github/CONTRIBUTING.md` should be updated as well.